frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/default-template.ts

```diff
@@ -55,6 +61,27 @@ export const resolveDefaultVMTemplate = (params: DefaultVMLikeEntityParams): Tem
         .asResource(),
     });
 
+  const cloudInitVolume = vm.getCloudInitVolume();
+
+  if (cloudInitVolume) {
+    const cloudInitHelper = new CloudInitDataHelper(cloudInitVolume.cloudInitNoCloud);
+    if (cloudInitHelper.hasKey(CloudInitDataFormKeys.PASSWORD)) {
+      cloudInitHelper.set(CloudInitDataFormKeys.PASSWORD, generateCloudInitPassword());
+      vm.removeStorage(CLOUDINIT_DISK).prependStorage({
```

Comment:
we are using the original cloud init storage with all the attributes. We don't have to replace it and we can just modify the password.